File: backend/onGo-domain/src/main/kotlin/com/ongo/domain/workspace/Workspace.kt
package com.ongo.domain.workspace

import java.time.LocalDateTime

data class Workspace(
    val id: Long? = null,
    val ownerId: Long,
    val name: String,
    val slug: String,
    val description: String? = null,
    val logoUrl: String? = null,
    val createdAt: LocalDateTime? = null,
    val updatedAt: LocalDateTime? = null,
)

File: frontend/src/types/workspace.ts
export interface Workspace {
  id: number
  ownerId: number
  name: string
  slug: string
  description: string | null
  logoUrl: string | null
  memberCount: number
  createdAt: string | null
}

export interface CreateWorkspaceRequest {
  name: string
  slug: string
  description?: string
}

export interface UpdateWorkspaceRequest {
  name?: string
  slug?: string
  description?: string
  logoUrl?: string
}
